digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1792,7 +1862,9 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
 
   }
 
-  private def getProjectsWaitingForTR: Seq[Long] = {
+  private def getProjectsWaitingForTR: Seq[Long]
```

Comment:
A bit way too much of spacing.

Reply:
auto-formatters... what's the standard in this project?